marienfressinaud/FreshRSS
# CURLOPT parameters per feed

Closes #3271

Changes proposed in this pull request:

 - proxy settings per feed
 - cookie settings per feed

How to test the feature manually:

1. set a proxy or cookie
2. check if the result changes

Pull request checklist:
